src/main/java/com/dev/boot/bankservice/model/Account.java

```diff
@@ -0,0 +1,30 @@
+package com.dev.boot.bankservice.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "accounts")
+public class Account {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true, nullable = false)
+    private String accountNumber;
+    @Enumerated(EnumType.STRING)
+    private Currency currency;
+    private BigDecimal balance;
+    private boolean isActive;
+    @ManyToOne
+    private User user;
```

Comment:
again, let's exclude this field from being displayed in toString()
we can set this with the lombok annotation